src/routes/ProductsRoute.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from 'express'
+import { getProducts } from '../controllers/ProductsController'
+// import {
+// } from '../controllers';
+// import { userAuth, checkInterviewer } from '../middlewares/auth';
+
+const router: Router = Router()
+
+router.get('/products', getProducts)
```

Comment:
this could be '/allproducts' endpoint to request all products since we gonna name the one in the index file 'products'